src/pages/media.tsx

```diff
@@ -1,172 +1,21 @@
-    resource: {
-      date: "5 Oct 2020",
-      time: "4pm - 4.30pm (SGT)",
-      eventLink: "https://www.swift.com/swift-at-sibos/joining-forces-trade-digitisation",
-    },
-  },
-  {
-    title: "Maritime Perspective Series: Digital Connectivity and Data Standards",
-    description:
-      "Organised by the Maritime Port Authority of Singapore (MPA), the panel discussed about what the future of global trade with common data standards will look like and what we need to do to get there.",
-    resource: {
-      date: "28 July 2020",
-      time: "4pm - 5pm (SGT)",
-      eventLink:
-        "https://www.mpa.gov.sg/web/portal/home/maritime-singapore/what-maritime-singapore-offers/maritime-perspectives-webinar-series/digital-connectivity-%26-data-standards",
-      watchLink: "https://www.youtube.com/watch?v=-rbi_k-URjw",
-    },
-  },
-  {
-    title:
-      "SBF Digital Economy Series: Trade Digitalisation - Lowering costs & improving efficiency through paperless trade",
-    description:
-      "This programme by SBF called on experts from IMDA and existing TradeTrust users to explain how businesses can benefit from TradeTrust.",
-    resource: {
-      date: "3 July 2020",
-      time: "3 - 4pm (SGT)",
-      eventLink: "https://www.sbf.org.sg/media-centre/publications-resources/webinars",
-      watchLink: "https://www.youtube.com/watch?v=NUTLtLso3-Y",
-    },
-  },
-  {
-    title: "Rethink, Retell and Restart",
-    description:
-      "RosettaNet Singapore and Global CIO Forum, GCF, hosted a webinar that offered insights across industries and countries, for CIOs to think of post-lockdown strategies in different markets.",
-    resource: {
-      date: "2 July 2020",
-      time: "3.30 - 5pm (SGT)",
-      eventLink: "https://globalcioforum.com/global-cio-forum-rosettanet-host-websummit-on-post-lockdown-strategies/",
-    },
-  },
-  {
-    title: "The Journey to Paperless Trade: Industry Initiatives for Interoperability",
-    description:
-      "This event by BAFT and R3 explored different blockchain-based digital trade solutions and how global trade participants can use these solutions towards challenges in the global trade ecosystem.",
-    resource: {
-      date: "2 July 2020",
-      time: "9 - 10pm (SGT)",
-      eventLink: "https://www.baft.org/baft-education/e-learning/past-webinars",
-      watchLink: "https://www.youtube.com/watch?v=iRmaQV9HERw",
-      eventSlides:
-        "https://baft.org/docs/default-source/past-webinars/webinar-the-journey-to-paperless-trade.pdf?sfvrsn=6e9a20e1_2",
-    },
-  },
-];
+const getSortedByDateDesc = (items: any[]) => {
+  items.sort((a, b): number => {
+    return new Date(b.attributes.date).getTime() - new Date(a.attributes.date).getTime();
+  });
+  return items;
```

Comment:
might want to use `compareDesc` from date-fns instead of rolling your own

Reply:
ah ok